source/includes/changelogs/releases/100.7.1.rst

```diff
@@ -0,0 +1,27 @@
+.. _100.7.1-changelog:
+
+100.7.1 Changelog
+-----------------
+
+*Released 2023-05-24*
+
+This release fixes a few bugs and adds downloads for macOS 11 on ARM
+as well as RedHat Enterprise Linux 9 (x86 and ARM) and Amazon Linux
+2023 (x86 and ARM).
```

Comment:
I'm not sure if we're tracking this in a separate ticket, but the [Platform Support matrix](https://docs-mongodbcom-staging.corp.mongodb.com/database-tools/docsworker-xlarge/DOCS-16161/installation/installation/#platform-support) doesn't mention RHEL 9. Sounds like it probably should.